src/content/blog/2023-01-10-eslint-community-org.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Announcing ESLint's official Community organization
+teaser: We are happy to announce the official ESLint Community organization.
+tags:
+  - Community organization
+  - Updates
+authors:
+  - MichaelDeBoey
+categories:
+  - Announcements
+---
+
+In our
+["ESLint's 2022 year in review" blogpost](https://eslint.org/blog/2023/01/eslint-2022-year-review),
+[Nicholas](https://github.com/nzakas) already briefly mentioned it in the
+['Project kickoffs' section](https://eslint.org/blog/2023/01/eslint-2022-year-review#project-kickoffs),
+but I'm happy to now officially announce the all new official
+[ESLint Community GitHub organization](https://github.com/eslint-community).
+
+The strength of ESLint is in its ecosystem, and we had been searching for a way
+to better organize that ecosystem. As a result, we created the
+`@eslint-community` GitHub organization as a home for high-value projects in the
+ESLint ecosystem.
+
+As you can read in the
+['`@eslint-community` GitHub organization' RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org),
+the goal of this new org is to have a place where community members can help
+ensure widely depended upon ESLint related packages live and thrive & never fall
+out of maintenance.  
+We will make sure these packages stay up to date with newer ESLint releases, and
```

Comment:
I think we need to define who "we" is here. Maybe introduce the community team before this sentence?